### DISCUSSION TITLE
Section 5 - Lesson 12 - StatelessFuzz test result looks different than the video

### DISCUSSION THREAD

**Discussion by sharonphiliplima** [Cyfrin/security-and-auditing-full-course-s23#234] [General]
My stateless fuzz test results looks different than what is shown in the video
Date - Sept 4th, 2024

```
sharon@sharon-Inspiron-5559:~/sc-exploits-minimized$ forge test --mt testFuzzCatchesBugStateless
[⠒] Compiling...
No files changed, compilation skipped
proptest: Aborting shrinking after the PROPTEST_MAX_SHRINK_ITERS environment variable or ProptestConfig.max_shrink_iters iterations (set 0 to a large(r) value to shrink more; current configuration: 0 iterations)

Ran 1 test for test/invariant-break/StatelessFuzzCatches.t.sol:StatelessFuzzCatchesTest
[FAIL. Reason: panic: assertion failed (0x01); counterexample: calldata=0x7d8339960000000000000000000000000000000000000000000000000000000000000002 args=[2]] testFuzzCatchesBugStateless(uint128) (runs: 0, μ: 0, ~: 0)
Suite result: FAILED. 0 passed; 1 failed; 0 skipped; finished in 3.01ms (1.47ms CPU time)

Ran 1 test suite in 26.82ms (3.01ms CPU time): 0 tests passed, 1 failed, 0 skipped (1 total tests)

Failing tests:
Encountered 1 failing test in test/invariant-break/StatelessFuzzCatches.t.sol:StatelessFuzzCatchesTest
[FAIL. Reason: panic: assertion failed (0x01); counterexample: calldata=0x7d8339960000000000000000000000000000000000000000000000000000000000000002 args=[2]] testFuzzCatchesBugStateless(uint128) (runs: 0, μ: 0, ~: 0)

Encountered a total of 1 failing tests, 0 tests succeeded
```

[fuzz]
runs = 1000
seed = '0x2'

[invariant]
runs = 1000
depth = 32
fail_on_revert = false

Also it worked even when I had fuzz runs = 2, 1 and 0! 
How and why?

**Comment by EngrPips** [comment]
Hello @sharonphiliplima, What do you mean when you said `My stateless fuzz test results looks different than what is shown in the video`?

**Reply by sharonphiliplima** [reply]
Mine has `proptest: Aborting shrinking after the PROPTEST_MAX_SHRINK_ITERS environment variable or ProptestConfig.max_shrink_iters iterations (set 0 to a large(r) value to shrink more; current configuration: 0 iterations)` while the one shown in Patrick's video isn't

Is this something new?

**Reply by EngrPips** [reply]
I mean, it has been months ago since Patrick made the video, and the tools we use get updated from time to time. So I will understand if there are some little differences in the way things are configured or implemented.

**Reply by olahamid** [reply]
hello @sharonphiliplima so after following up with the last issue that was created in the error, the is the solution to it incase of future users. 
All i know about this is that it is a rust and cache issue,
step 1
 ```
foundryup
```
step 2
 ```
forge clean 
```
step 3 
```
forge test --match-test testFuzzCatchesStateless -vvvv
```

**Comment by EngrPips** [comment]
Interesting.
